x-pack/plugins/alerting/common/system_actions/v1.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema } from '@kbn/config-schema';
+import { RuleActionTypes } from '../rule';
+
+export const systemActionSchema = schema.object({
```

Comment:
@JiaweiWu Let me know if you want me to put the schema in a different place. It is the same in the routes and the rules client.